Dockerfile

```diff
@@ -31,14 +31,7 @@ RUN if [ "${TAGS}" = "fcos" ]; then \
     elif [ "${TAGS}" = "scos" ]; then \
     # rewrite image names for scos
     sed -i 's/rhel-coreos/centos-stream-coreos-9/g' /manifests/*; fi && \
-    # pin nmstate to 2.2.9 until we update to https://github.com/openshift/machine-config-operator/pull/3720 \
-    . /etc/os-release; \
-    NMSTATE_PKG=nmstate-2.2.9-6.rhaos4.14.el8; \
```

Comment:
@cgwalters I don't recall fully, but was there a reason to use el8 package instead of el9 ? or this is something we have been carrying because during the time we pinned it, we had only el8 built nmstate.